From ceedc865c3adaa6bad36f621a29bac5c8fd36346 Mon Sep 17 00:00:00 2001
From: Julian Andres Klode <jak@jak-linux.org>
Date: Sun, 26 Nov 2017 12:30:48 +0100
Subject: [PATCH] Install stuff to $(PACKAGE) in $(libexecdir), not $(libdir)

This does not really make a huge difference, probably, but it
is infinitely more correct.
---
 Makefile.autosetup | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

--- a/Makefile.autosetup
+++ b/Makefile.autosetup
@@ -38,6 +38,7 @@
 docdir=		@PKGDOCDIR@
 mandir=		@mandir@
 libdir=		@libdir@
+libexecdir=		@libexecdir@
 sysconfdir=	@SYSCONFDIR@
 
 # targets for specific subdirectories
@@ -272,9 +273,9 @@
 	for f in $(BINFILES); do \
 		$(INSTALL) $$f $(DESTDIR)$(bindir)/`basename $$f`; \
 	done
-	$(MKDIR_P) $(DESTDIR)$(libdir)/$(PACKAGE)
+	$(MKDIR_P) $(DESTDIR)$(libexecdir)/$(PACKAGE)
 	for f in $(LIBBINFILES); do \
-		$(INSTALL) $$f $(DESTDIR)$(libdir)/$(PACKAGE)/`basename $$f`; \
+		$(INSTALL) $$f $(DESTDIR)$(libexecdir)/$(PACKAGE)/`basename $$f`; \
 	done
 
 # uninstall
@@ -284,7 +285,7 @@
 		$(RM) $(DESTDIR)$(bindir)/`basename $$f`; \
 	done
 	for f in $(LIBBINFILES); do \
-		$(RM) $(DESTDIR)$(libdir)/$(PACKAGE)/`basename $$f`; \
+		$(RM) $(DESTDIR)$(libexecdir)/$(PACKAGE)/`basename $$f`; \
 	done
 
 # distclean
